storage/integration_test.go

```diff
@@ -1069,13 +1069,42 @@ func testObjectIterator(t *testing.T, bkt *BucketHandle, objects []string) {
 	if !ok {
 		t.Errorf("ObjectIterator.Next: %s", msg)
 	}
+	m := make(map[string][]*ObjectAttrs)
```

Comment:
These look like the right things to test. Could you move this into its own testing function, call it `testObjectIteratorWithOffset` or similar? That way we can easily see that a failure is due to offset issues.

Reply:
Done